src/components/Explore/TokenRow.tsx

```diff
@@ -0,0 +1,221 @@
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 12px 8px;
+  gap: 10px;
+
+  width: 107.5px;
+  height: 60px;
+`
+const PercentChangeContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  padding: 14px 8px;
+  gap: 10px;
+
+  width: 107.5px;
+  height: 60px;
+`
+
+const PercentChangeContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+`
+
+const MarketCapContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 12px 8px;
+  gap: 10px;
+
+  width: 107.5px;
+`
+const VolumeContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 12px 8px;
+  gap: 10px;
+
+  width: 107.5px;
+`
+const SparkLineContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 16px 24px;
+  gap: 10px;
+
+  width: 172px;
+`
+const SparkLineImg = styled.div`
+  max-width: 124px;
+  max-height: 28px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  transform: scale(1.2);
+`
+
+const SwapContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 16px 4px;
+  gap: 10px;
+
+  width: 62px;
+`
+
+const SwapButton = styled.button`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding: 8px;
+  gap: 6px;
+
+  width: 54px;
+  height: 32px;
+
+  background: ${({ theme }) => theme.primary2};
+  border-radius: 12px;
+  border: none;
+  color: ${({ theme }) => theme.white};
+`
+
+const TokenSymbol = styled.span`
+  color: ${({ theme }) => theme.text3};
+`
+const ArrowContainer = styled.div`
+  padding: 0px 0px 0px 4px;
+`
+
+/* formats price with appropriate string */
+function priceFormatter(num: number): string {
+  if (num >= 1000000000) {
+    return (num / 1000000000).toFixed(1).replace(/\.0$/, '') + 'G'
+  }
+  if (num >= 1000000) {
+    return (num / 1000000).toFixed(1).replace(/\.0$/, '') + 'M'
+  }
+  if (num >= 1000) {
+    return (num / 1000).toFixed(1).replace(/\.0$/, '') + 'K'
+  }
+  return num.toString()
+}
+
+export default function TokenRow({
+  key,
+  tokenAddress,
+  data,
+  listNumber,
+  timePeriod,
+}: {
+  key: string
+  tokenAddress: string
+  data: TokenData
+  listNumber: number
+  timePeriod: TimePeriod
+}) {
+  const token = useToken(tokenAddress)
+  const tokenName = token?.name
+  const tokenSymbol = token?.symbol
+  const tokenData = data[tokenAddress]
+  // TODO: write favorited hook
+  // TODO: remove magic number colors
+  const favorited = false
+  return (
+    <TokenRowWrapper key={key}>
+      <FavoriteContainer>{favorited ? <Heart size={15} /> : <Heart size={15} />}</FavoriteContainer>
```

Comment:
is it supposed to be a diff color?

Reply:
Oh yes this is. I couldn't find a filled in heart from feather react icons so I was trying to find some workaround